src/modules/app/routes/categories.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+import categoriesFetcher from '../../endpoint/endpoints/categories'
+import { createAction } from 'redux-actions'
+
+import type { Dispatch, GetState } from 'redux-first-router/dist/flow-types'
+
+export const CATEGORIES_ROUTE = 'CATEGORIES'
+
+export const goToCategories = (city: string, language: string, categoryPath: ?string) =>
+  createAction(CATEGORIES_ROUTE)({city, language, categoryPath})
+
+/**
+ * CategoriesRoute, matches /augsburg/de*
+ * @type {{path: string, thunk: function(Dispatch, GetState)}}
+ */
+export const categoriesRoute = {
```

Comment:
Do we have a type for a route?

Reply:
Currently the type is documented in a comment. This should be "documented" by using flow.